MinecraftClient/ChatBot.cs

```diff
@@ -662,6 +664,45 @@ public bool GetInventoryEnabled()
         {
             return Handler.GetInventoryEnabled();
         }
+        public Dictionary<int, Container> GetInventories()
+        {
+            return Handler.GetInventories();
+        }
+        /// <summary>
+        /// Use item in hand
+        /// </summary>
+        protected void UseItemInHand()
+        {
+            Handler.UseItemOnHand();
+        }
+        /// <summary>
+        /// start Sneaking
+        /// </summary>
+        protected bool Sneak()
+        {
+            return SendAction(Protocol.ActionType.StartSneaking);
+        }
+        /// <summary>
+        /// Sends UnSneak
+        /// </summary>
+        protected bool UnSneak()
```

Comment:
Since Sneaking is either ON or OFF, maybe define the following function instead of 2 functions?
```C#
protected bool Sneak(bool on)
```